action.yml

```diff
@@ -13,12 +13,20 @@ inputs:
   command:
     description: 'Command to run.'
     required: false
-    default: 'cargo kani --workspace'
+    default: 'cargo-kani'
 
 runs:
-  using: 'docker'
-  image: docker://ghcr.io/model-checking/kani-ubuntu-20.04:0.11.0
-  args:
-    - bash
-    - '-c'
-    - 'set -e; export HOME=/root USER=$(id -nu) PATH=/root/.cargo/bin:$PATH; source $HOME/.bashrc; ${{ inputs.command }}'
+  using: "composite"
+  steps:
+    - name: Install Kani
+      shell: bash
+      run: |
+        export KANI_VERSION="0.11.0"; \
```

Comment:
Should we retrieve the version from the repository instead? I was also wondering if we should allow users to choose the version they want to use so they can control when to update Kani.

Reply:
My thought was action version matches kani-version. So we have kani-action@0.11 matches kani 0.11